### ISSUE TITLE
Unable to deploy to Heroku, keep getting Cannot find module 'SendBird' error.

### ISSUE THREAD

**Issue by AlexSav1** [smilefam/SendBird-SDK-JavaScript#78]
Hello,

I am trying to use the SendBird SDK version 3.0.63 in my angular application, everything works great when running it locally but so far I've been unable to deploy it to Heroku. I keep getting error: Cannot find module 'SendBird'.

I am importing the module into my service as suggested using

`import * as SendBird from 'SendBird';`
`var sb = new SendBird({'appId': 'APP_ID'});`

I've tried setting 
`"allowSyntheticDefaultImports": true`
in tsconfig.json but no luck

I've also tried several different methods of importing the module to no avail.

**Comment by Alchemist85K** [comment]
Hi @AlexSav1 .
Thank you for your interest in SendBird SDK.
I'll let you know after I'll reproduce this issue.

**Comment by Alchemist85K** [comment]
@AlexSav1 
I tried to reproduce this problem and tried it with the version 3.0.63 you used, but it worked successfully.
I created a project using @angular/cli and modified just main.ts as follows.

```
import { enableProdMode } from '@angular/core';
import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';

import { AppModule } from './app/app.module';
import { environment } from './environments/environment';

import * as SendBird from 'SendBird';

if (environment.production) {
  enableProdMode();
}

platformBrowserDynamic().bootstrapModule(AppModule)
  .then(() => {
    console.log('init SendBird');
    const appId = '9DA1B1F4-0BE6-4DA8-82C5-2E81DAB56F23'; // DEMO SAMPLE APP_ID
    var sb = new SendBird({ appId });
    sb.connect('test_user', function(user, err) {
      console.log('[connect func] res: ', user, ' err: ', err);
    })
  })
  .catch(err => console.log(err));
```

**Comment by AlexSav1** [comment]
Ok so I got it working, deploy kept failing trying to run 'ng build -prod' in my postinstall script. It kept saying cannot find module but only when trying to deploy. When I would run ng build locally everything went fine.

It appears to be a case-sensitive issue,

I changed 
`import * as SendBird from 'SendBird';`
to 
`import * as SendBird from 'sendbird';`

so it matches the name of the node module exactly and now everything builds and deploys fine. My local machine is not using a case sensitive file system so thats why it would work locally.

Anyway thanks for responding so quickly, the SDK is awesome I'm looking forward to using it more.
